src/crm/dto/contact-search.dto.ts

```diff
@@ -6,17 +6,12 @@ export class ContactSearchDto extends SearchDto {
   email?: string;
   phone?: string;
 
-  @Type(() => Number)
-  @IsArray()
-  @IsNumber({}, { each: true })
-  @IsOptional()
-  cellGroups?: number[];
-
```

Comment:
Am a bit curious why you removed the validation logic

Reply:
I have restored the validation logic and made some other changes at the processing of data at the client side.